### PR TITLE
Memtest portage

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

**Comment by miczyg1** [comment]
I have additionally cleaned up some code related to iPXE.

**Comment by pietrushnic** [comment]
@miczyg1  great did you exercised this in gitlab-Ci pipline?

**Comment by miczyg1** [comment]
@pietrushnic not yet. Tested only compilation on my PC. I am currently working to use it with pce-fw-builder and CI.

**Comment by pietrushnic** [comment]
@miczyg1 I'm fine with current changes, but let's wait until it would be proved to work with CI, then we can proceed with merging.

**Comment by miczyg1** [comment]
@pietrushnic legacy can be now built the same way as mainline (copy config and run make). CI pipeline works as well after small change to YAML.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ build_crosscompiler:
     - repo init -u https://github.com/pcengines/release_manifests.git -b ${REPO_REVISION}
     - repo sync --force-sync
     - cp ${CI_PROJECT_DIR}/configs/pcengines_${PLATFORM}.config ${CI_PROJECT_DIR}/apu2_fw_rel/apu2/coreboot/.config
-    - ${RELEASE_DIR}/apu2/apu2-documentation/scripts/build_release_img.sh build
+    - ${RELEASE_DIR}/apu2/apu2-documentation/scripts/build_release_img.sh build-ml
```
Comment (pietrushnic) [inline .gitlab-ci.yml]: @miczyg1 I'm not sure how this improve situation.

**Pull request merged**